jdaviz/configs/imviz/plugins/catalogs/catalogs.vue

```diff
@@ -53,5 +53,11 @@
        <span style='padding-left: 4px' v-if="results_available">{{number_of_results}}</span>
     <v-row>
 
-  </j-tray-plugin>
-</template>
+    <j-tray-plugin
+        :keep_active.sync="keepActive"
+        :popout_button="popoutButton"
+        :scroll_to.sync="scroll_to">
+        <jupyter-widget :widget="table_widget"></jupyter-widget>
+
+    </j-tray-plugin>
```

Comment:
is the `j-tray-plugin` to try to fix the layout issues @rosteen mentioned?  That is only meant to be wrapped around the very top-level of the plugin itself... maybe wrapping in a `v-row` will do what you want?

Reply:
I just did a recent commit where I have the table displaying at the top and at the bottom. The one at the top displays correctly while the one on the bottom does not. I am unsure of what's going wrong.

Reply:
ok - I will take a look in the next few hours and see if I can help debug.  Definitely seems to be a weird case!